src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.tsx

```diff
@@ -56,6 +57,7 @@ export const ItemConfiguration = () => {
       isTextInputOptionRequired: true,
       minNumber: DEFAULT_MIN_NUMBER,
       maxNumber: DEFAULT_MAX_NUMBER,
+      sliderOptions: [getEmptySliderOption()],
```

Comment:
pls make this value not to be compulsory, and appears only if the user is chosen the slider or multiple slider input type, also remove this value if the user is chosen a different from these input types

Reply:
done